templates/zerver/help/index.md

```diff
@@ -143,7 +143,7 @@ as a “**realm**”.
 * Deactivate or reactivate a user
 * Deactivate or reactivate a bot
 * [Make a user an administrator](/help/make-a-user-an-administrator)
-* Change a user's name
+* [Change a user's name](/help/change-a-user's-name)
```

Comment:
OK, I'm not sure about this, but are apostrophes allowed in a URL name? You might have to check in your environment to see if the link works.